app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+class InvitesController < ApplicationController
+  before_action :check_user_registration, :authenticate_user!
+  before_action :fetch_current_invite, only: [:accept_invite]
+  before_action :check_user_organization, only: %i[index create]
```

Comment:
@JuanCrg90 when using arrays of symbols is recommended to create them using `%i[element1 element2]` without `:` and `,`.